lib/ansible/modules/cloud/vmware/vmware_object_role_permission.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Derek Rushing <derek.rushing@geekops.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vmware_object_role_permission
+short_description: This module can be used to manage object permissions on the given host.
```

Comment:
Can you interchange short_description and description ?

Reply:
Swapped these out